Command/DoctrineCommand.php

```diff
@@ -99,6 +99,10 @@ public static function configureMigrations(ContainerInterface $container, Config
             $configuration->setCustomTemplate($container->getParameter('doctrine_migrations.custom_template'));
         }
 
+        if (method_exists($configuration, 'isDatabasePlatformChecked')) {
```

Comment:
Shouldn't this check for `setCheckDatabasePlatform()`?

Reply:
I don't know flow for similar issues, so I just follow the same style as above (custom template).

I can change check to `setCheckDatabasePlatform` if you prefer.

Reply:
The one above is also using the checked method name `getCustomTemplate()` in the if-block. There would still be a `empty($configuration->getCustomTemplate())` even if the method was available in all versions of `$configuration`.

If you check for an existing method `isDatabasePlatformChecked()`, shouldn't this method be called and checked for `true` or `false` in the if-block too? If `isDatabasePlatformChecked()` is not relevant to be called, you have to check if the method `setCheckDatabasePlatform()` exists.

Reply:
Changed to check `set-` method.

Reply:
You removed the `method_exists` for `setCheckDatabasePlatform()`. Did you find out that checking if this method exists isn't needed?

Reply:
It's not needed now, because I increase minimal `doctrine/migrations` version to 2.1